guide/using-the-e-puck-robot.md

```diff
@@ -300,3 +275,51 @@ This error can be fixed simply by installing the following packets from a termin
 ```sh
 sudo apt-get install gcc-4.6-base:i386 libgcc1:i386 libc6:i386
 ```
+
+### Samples
+
+Several simulation examples are located in the "WEBOTS\_HOME/projects/robots/e-puck/worlds" directory of the Webots distribution.
+
+The directory "WEBOTS\_HOME/projects/samples/curriculum" contains a rich collection of simulations involving the e-puck robot.
```

Comment:
I'm wondering it the Curriculum would not deserve a section (h2) rather. It seems this important URL is missing: `https://en.wikibooks.org/wiki/Cyberbotics'_Robot_Curriculum`